<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSuchen_Ersetzen</class>
 <widget class="QDialog" name="DlgSuchen_Ersetzen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>311</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Suchoptionen</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="chkRueckwaerts">
       <property name="text">
        <string>Rückwärtssuche</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkGrossKlein">
       <property name="text">
        <string>Groß- / Kleinschreibung beachten</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkGanzeWorte">
       <property name="text">
        <string>Nur ganze Wörter berücksichtigen</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>381</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Suchbegriff</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leSuchbegriff"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnSuchen">
      <property name="text">
       <string>&amp;Suchen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>471</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnErweitern">
      <property name="text">
       <string>Erweitern</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Schließen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>381</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ersetzen durch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leErsetzenDurch"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnErsetzen">
        <property name="text">
         <string>Ersetzen</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnAllesErsetzen">
        <property name="text">
         <string>Alles Ersetzen</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leSuchbegriff</tabstop>
  <tabstop>chkRueckwaerts</tabstop>
  <tabstop>chkGrossKlein</tabstop>
  <tabstop>chkGanzeWorte</tabstop>
  <tabstop>btnSuchen</tabstop>
  <tabstop>btnErweitern</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>leErsetzenDurch</tabstop>
  <tabstop>btnErsetzen</tabstop>
  <tabstop>btnAllesErsetzen</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>DlgSuchen_Ersetzen</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
